test/runtests.jl

```diff
@@ -8,3 +8,42 @@ using Test
   @test inv(Identity() → Identity()) == Identity()
   @test (Identity() → Identity()) == Identity()
 end
+
+@testset "Testing Fallbacks" begin
```

Comment:
Please remove this test set and move it to inside the single test set of the package

Reply:
Done.

Reply:
It is not, it is just because all test sets should live under the package set set. Also, this package is so small that a single test set is enough.